From cb282869903015cd86a7541be138c00bd03fa88d Mon Sep 17 00:00:00 2001
From: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
Date: Wed, 22 Jan 2020 19:13:21 +0530
Subject: [PATCH 0960/1851] net: emaclite: In kconfig remove arch dependency

commit 7ff818324d9eb929ff7c546db79cf444412da899 from
https://github.com/Xilinx/linux-xlnx.git

To enable xilinx_emaclite driver support on zynqmp ultrascale platform
(ARCH64) remove the obsolete ARCH dependency list. Also include HAS_IOMEM
dependency to avoid compilation failure on architectures without IOMEM.

Sanity build test done for microblaze, zynq and zynqmp ultrascale platform.

Signed-off-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
Link: https://www.spinics.net/lists/arm-kernel/msg776138.html
State: waiting (net)
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/net/ethernet/xilinx/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/xilinx/Kconfig b/drivers/net/ethernet/xilinx/Kconfig
index 2db9e9d7e0f6..0692dd12721d 100644
--- a/drivers/net/ethernet/xilinx/Kconfig
+++ b/drivers/net/ethernet/xilinx/Kconfig
@@ -19,6 +19,7 @@ if NET_VENDOR_XILINX
 config XILINX_EMACLITE
 	tristate "Xilinx 10/100 Ethernet Lite support"
 	select PHYLIB
+	depends on HAS_IOMEM
 	---help---
 	  This driver supports the 10/100 Ethernet Lite from Xilinx.
 
-- 
2.31.1

